wireapp/kalium
# chore: add archived field to conversations table (WPB-4471)

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

We were retrieving `otr_archived` and `otr_archived_ref` from the API but not persisting in the database.

### Solutions

Add `otr_archived` and `otr_archived_ref` to conversation table when getting from API.



### Testing

#### Test Coverage (Optional)


#### How to Test

Currently the only way to test is:
- Fresh install/login
- check from App Inspection that you have both fields in the `Conversation` table and if you have any archived conversation, it should be represented as such (1 = true, 0 = false)
